epub.go

```diff
@@ -100,6 +98,7 @@ img {
 // Epub implements an EPUB file.
 type Epub struct {
 	sync.Mutex
+	*http.Client
```

Comment:
do we really need to keep a reference to the http Client in the whole application when it's only used in one place (httpHandler)? could we just create the client in the place where we need it and use the timeout as recommended by the article you shared?

Reply:
It is cheap to add it here, and it allows setting a custom client for the whole Epub. 
IMHO, it allows decoupling the infrastructure layer and the business layer saying something like _this document is using this infrastructure transport layer_.

On top of that, I guess that eventually the `addMedia` method should be changed to add a `context.Context` as first argument to allow cancelation mechanism (see https://go.dev/blog/context). Some people are using the library concurrently, therefore this cancelation mechanism would bring some value.
If we do that, we will end with a lot of parameters which will make testing a little bit harder.

WDYT?